src/main/java/io/fabric8/maven/docker/SaveMojo.java

```diff
@@ -39,6 +39,12 @@
 	@Parameter(property = "docker.skip.save", defaultValue = "false")
 	private boolean skipSave;
 
+	@Parameter(property = "docker.save.attach", defaultValue = "false")
+	private boolean saveAttach;
+
+	@Parameter(property = "docker.save.classifier")
+	private String saveClassifier;
```

Comment:
I wonder whether we could combine both properties (as I'm always trying to keep the configuration surface small). What's about having a single 'saveAttach' string property, which can be 'true/false' (then a default classifier is select for 'true') or any other value in which case it enables the featue and uses that classifier ?

Reply:
We can, although I always worry about making something that looks boolean, but isn't, because of [previous mockery](https://thedailywtf.com/articles/What_Is_Truth_0x3f_).

I could make specifying a non-empty `saveClassifier` mean "attach with this classifier", and not provide the default logic, since by the time you write `<saveAttach>true</saveAttach>`, you probably could have written `<saveClassifier>archive</saveClassifier>`, and we could use %a for the alias (as in `<saveClassifier>archive-%a</saveClassifier>` or -Ddocker.save.classifier=archive-%a)?

Reply:
> We can, although I always worry about making something that looks boolean, but isn't, because of [previous mockery](https://thedailywtf.com/articles/What_Is_Truth_0x3f_).
> 
> I could make specifying a non-empty `saveClassifier` mean "attach with this classifier", and not provide the default logic, since by the time you write `<saveAttach>true</saveAttach>`, you probably could have written `<saveClassifier>archive</saveClassifier>`, and we could use %a for the alias (as in `<saveClassifier>archive-%a</saveClassifier>` or -Ddocker.save.classifier=archive-%a)?

Fair enough. I think having no default name is fine as adding one is trivial (also saves the user a documentation lookup for finding out what the default would be).